### DISCUSSION TITLE
2008/04/18/%E5%88%9B%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA-msn-%E7%9A%84%E7%BE%A4%E7%BB%99-labview

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#216] [Announcements]
# 2008/04/18/%E5%88%9B%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA-msn-%E7%9A%84%E7%BE%A4%E7%BB%99-labview

从来没用过QQ，据说它的群功能很好用，但是msn没这个功能。后来有个第三方公司和msn合作，提供了类似群的功能。我听说后就跑上去给LabVIEW建了一个。可是当时没几个人，所以一直都没用。现在有半年过去了，再跑上去一看，成员居然多起来了。而且经常有人发言，不像以前那样冷清了。

https://blog.qizhen.xyz/2008/04/18/%E5%88%9B%E5%BB%BA%E4%BA%86%E4%B8%80%E4%B8%AA-msn-%E7%9A%84%E7%BE%A4%E7%BB%99-labview

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年04月20日 03:23 编辑
嘿嘿，在space的统计一栏中会列出读者是从哪跳转到我这里的，我经常跟踪回去看看大家为啥来访问我的网页。有些工具可以列出访问来源的IP

回复
Bluebaby
2008年04月20日 01:58 编辑
你怎么知道我看你的SPACE了?恐怖啊!

回复
